graspit_interface/Grasp[] grasps
float64[] energies
string search_energy
int32 num_planning_steps
float64[] is_reachable
float64[] lift_success
graspit_interface/Grasp[] moveit_grasps